duckduckgo/Android
# Add logos for Google and Meta child entities.

Task/Issue URL: https://app.asana.com/0/488551667048375/1202848775485203/f

### Description
Adds icons for Google and Facebook child entities (added in the next blocklist update).

### Steps to test this PR
1. Switch to the 'beta' blocklist (change 'v3' to 'beta' [here](https://github.com/duckduckgo/Android/blob/develop/app/src/main/java/com/duckduckgo/app/trackerdetection/api/TrackerListService.kt#L23)).
2. Visit test sites. Verify that for the new entities the correct icon is shown:
  - "Google Ads (Google)": youtube.com, android.com
  - "Google Analytics (Google)": blogger.com
  - "Youtube (Google)": https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/ (in not blocked section).
  - "Instagram (Facebook)": https://www.cinemablend.com/movies/see-the-four-hobbits-from-lotr-reuniting-at-a-restaurant-and-totally-making-the-day-of-a-nearby-fan
